### DISCUSSION TITLE
Task list progress should be reflected in Milestones

### DISCUSSION THREAD

**Discussion by nitnelave** [community/community#4365] [Projects and Issues]
I use tracking issues with taks lists for the milestones, so I'll have one issue per main area of work, with a task list. Progress on the milestone only counts completed issues, not tasks, so it's very coarse-grained.

I'd like an option to be able to reflect a task list as progress on a specific issue.

**Comment by nitnelave** [comment]
Otherwise, love the feature!

**Comment by ZakaHaceCosas** [comment]
I was about to suggest the same idea! Milestones are already good, but instead of reflecting 0/2 issues, whenever an issue has tasks, milestones should show the 4/31 (example) task count as well, and use that for the progress bar. +1 for this!
